packages/frontend/src/pages/EventLogs/EventLogsPage.tsx

```diff
@@ -109,7 +109,7 @@ const EventLogsPage = (): JSX.Element => {
           </div>
 
           {/* Search */}
-          <div className="w-5/12 mt-5 mb-5 mr-4">
+          <div className="w-5/12 mt-5 mb-5 mr-4 h-[42px]">
```

Comment:
Can we use tailwind's built-in sizing classes instead of fixed pixel sizes?

Reply:
We just want to ensure all the inputs are the same height. If that can be achieved without specifying a fixed height, all the better.